commons/tool/src/webapp/js/commons.js

```diff
@@ -110,13 +110,13 @@ commons.switchState = function (state, arg) {
             }
 
             editor.click(function (e) {
-
```

Comment:
How dare you remove my classic blank like like that? The audacity :)

Reply:
I'm sorry! Feel free to add it again :)